### PR TITLE
Check enabled page is loading after uninstalling operator

### PR DESCRIPTION
**Pull request by rkamble5**
Signed-off-by: rkamble <rkamble@redhat.com>

### CODE REVIEW

**Comment by rkamble5** [comment]
Tested Successfully here :-https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1022/
it failed because the enabled page showing "Error loading components" it should not show

**Comment by rkamble5** [comment]
Tested Successfully here :-https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1037/
it failed because the enabled page showing "Error loading components" it should not show

**Review by bdattoma — APPROVED** [review]

File: tests/Tests/400__ods_dashboard/400__ods_dashboard.robot [hunk 1]
```diff
@@ -390,3 +403,37 @@ Verify Anaconda Success Message Based On Version
...
+
+Uninstall Operator And Check Enabled Page Is Rendering
+    [Documentation]    Uninstall Operator And Check Enabled Page(ODS) Is Rendering, Not shwoing "Error loading components"
+    [Arguments]     ${operator_name}    ${operator_appname}
+    Open Installed Operators Page
+    Uninstall Operator    ${operator_name}
+    Launch Dashboard   ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    ...   ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
+    Page Should Not Contain    Error loading components
+    Close All Browsers
```
Comment (bdattoma) [inline tests/Tests/400__ods_dashboard/400__ods_dashboard.robot:423]: maybe this is not really needed, because the Teardown already closes all the browsers

Comment (rkamble5) [reply tests/Tests/400__ods_dashboard/400__ods_dashboard.robot]: done

**Review by bdattoma — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]
lgtm

**Comment by bdattoma** [comment]
@rkamble5 plz update polarion and its corresponding Jira issues

**Pull request merged**